python/lsst/sims/maf/plots/neoDetectPlotter.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from .plotHandler import BasePlotter
+from matplotlib.patches import Ellipse
+
+__all__ = ['NeoDetectPlotter']
+
+class NeoDetectPlotter(BasePlotter):
```

Comment:
I'm okay with leaving this in the core of maf (rather than maf_contrib), but can I be picky and say instead of "NeoDetect" we change all references to "NeoDetect" to be something like "NeoDistance" or NeoMaxDistance? 

My motivation for doing so is just to make sure that anyone coming in and looking for metrics or anything else related to a NEO detection related to orbits doesn't get confused and sidetracked into this and then think that this is how LSST thinks to characterize its NEO detection rate. NeoMaxDistance or NeoDistance seems pretty clearly about the distances you could detect an object to, rather than the overall detection rate.